### PR TITLE
client: Update help

### PR DESCRIPTION
**Pull request by okurz**
Add help for all options, better examples and a good introduction text.

### CODE REVIEW

**Comment by okurz** [comment]
Followup to our discussion in https://github.com/os-autoinst/openQA/pull/842

File: script/client [hunk 1]
```diff
@@ -44,6 +40,21 @@ load get/post parameters from a json file. For example
    "VERSION" : "26",
 }
 
+=item B<--apibase>
+
+Set API base URL component, default: '/api/v1'
+
+=item B<--json>
+
+Output raw JSON instead of prettified perl structures.
```
Comment (coolo) [inline script/client]: is there cooked JSON too?

**Comment by okurz** [comment]
> is there cooked JSON too?

I expected "raw" JSON as in "not prettyfied", updated, please check again.

**Comment by okurz** [comment]
updated to lower coverage threshold (after the screenshot table PR which lowered coverage)

**Comment by aaannz** [comment]
Still 0.2% short.

**Comment by aaannz** [comment]
Interesting that coverage reported by travis differ from coverage posted by coveralls.

**Comment by okurz** [comment]
and also it seems conducting the tests locally I get 71.0% but travis only reaches 70.8% I will leave the coverage threshold update PR for another time

**Pull request merged**